src/views/Map/AgentMAp.js

```diff
@@ -0,0 +1,173 @@
+
+import React from 'react'
+// import { AppSwitch } from '@coreui/react'
+import { Row } from 'reactstrap';
+
+// let map, infoWindow, marker, markers, geoCodeValue;
+let map, pos, panorama, infoWindow;
+
+class AgentMap extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.googleMapRef = React.createRef();
+    this.initMap = this.initMap.bind(this);
+    this.addRoute = this.addRoute.bind(this);
+    this.handleLocationError = this.handleLocationError.bind(this);
+    this.processSVData = this.processSVData.bind(this);
+  }
+
+  componentDidMount() {
+    let googleScript = document.createElement('script');
+    // API key invalid, add yours or contact @johnerry on git
+    googleScript.src = 'https://maps.googleapis.com/maps/api/js?key=AIzatSyBKHtdFymwut91lTBTQBDgltrFaXPD6TxY2g';
+    googleScript.async = true;
+    window.document.body.appendChild(googleScript);
+    googleScript.addEventListener('load', this.initMap);
+  }
+
+  initMap() {
+    let google = window.google;
+    let addRoute = this.addRoute;
+    const handleLocationError = this.handleLocationError;
+    let processSVData = this.processSVData;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['processSVData' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)